cli/src/commit_templater.rs

```diff
@@ -1027,14 +1027,8 @@ impl RefNamesIndex {
         }
     }
 
-    #[allow(unknown_lints)] // XXX FIXME (aseipp): nightly bogons; re-test this occasionally
-    #[allow(clippy::manual_unwrap_or_default)]
     pub fn get(&self, id: &CommitId) -> &[Rc<RefName>] {
-        if let Some(names) = self.index.get(id) {
-            names
-        } else {
-            &[]
-        }
+        self.index.get(id).map_or(&[], |names| names)
```

Comment:
Optional: add a comment why `unwrap_or` does not work here? `|names| names` does look slightly awkward.

Also, I wouldn't mind understanding the issue better myself.

It's fine if you don't feel like it.

Reply:
Ok, I added type annotation `names: &Vec<_>` to better explain `|x| x`.